aosp_diff/preliminary/packages/providers/MediaProvider/03_0003-Handle-content-uris-without-id-in-checkUriPermission.patch

```diff
@@ -0,0 +1,120 @@
+From 15dc9f3496376832f8a1fe2f15d0f66c57248f0c Mon Sep 17 00:00:00 2001
+From: Ivan Chiang <chiangi@google.com>
+Date: Mon, 16 Aug 2021 15:13:32 +0800
+Subject: [PATCH] Handle content uris without id in checkUriPermission
+
```

Comment:
This is seen as a already applied patch.
Project ./packages/providers/MediaProvider
        Already applied         01_0001-Handle-content-uris-without-id-in-checkUriPermission.patch
        Applying                02_0002-Restrict-legacy-apps-to-insert-files-to-other-privat.patch
        Already applied         03_0003-Handle-content-uris-without-id-in-checkUriPermission.patch